backend/tests/services/knowledge/group_kb/test_group_kb_crud.py

```
@@ -0,0 +1,613 @@
+    test_group_reporter,
+    test_subgroup,
+)
+
+
+class TestCreateKnowledgeBase:
+    """Test knowledge base creation permissions."""
+
+    def test_owner_can_create_kb(
+        self,
+        test_db: Session,
+        test_group,
```

Ruff (0.15.2)

[error] Redefinition of unused `test_group` from line 34: `test_group` redefined here

(F811)